pkg/storage/system_intake.go

```diff
@@ -420,6 +421,25 @@ func (s *Store) FetchSystemIntakes(ctx context.Context) (models.SystemIntakes, e
 	return intakes, nil
 }
 
+// FetchSystemIntakesWithReviewRequested queries the DB for all open system intakes where user is requested
+func (s *Store) FetchSystemIntakesWithReviewRequested(ctx context.Context, userID uuid.UUID) ([]*models.SystemIntake, error) {
+	intakes, err := sqlutils.WithTransactionRet(ctx, s, func(tx *sqlx.Tx) ([]*models.SystemIntake, error) {
+		intakes := []*models.SystemIntake{}
+		err := namedSelect(ctx, s, &intakes, sqlqueries.SystemIntakeGRBReviewer.GetIntakesWhereReviewRequested, map[string]interface{}{
```

Comment:
```suggestion
		err := namedSelect(ctx, s, &intakes, sqlqueries.SystemIntakeGRBReviewer.GetIntakesWhereReviewRequested, args{
```

I think we can use `args` here as a shorthand!

Reply:
I've come full circle on these methods. Added in [596cb5d](https://github.com/CMSgov/easi-app/pull/2663/commits/596cb5d441877771f98003bff266232488d33fcc)